Framework/API/inc/MantidAPI/IndexProperty.h

```diff
@@ -40,7 +43,7 @@ namespace API {
   File change history is stored at: <https://github.com/mantidproject/mantid>
   Code Documentation is available at: <http://doxygen.mantidproject.org>
 */
-class MANTID_API_DLL IndexProperty : public Kernel::ArrayProperty<int> {
+class MANTID_API_DLL IndexProperty : public Kernel::ArrayProperty<int64_t> {
```

Comment:
Why `int64_t` we should never need this to be signed?

Reply:
`specnum_t` and `detid_t` are signed (the latter is not supported yet).